bindings/go/build_dynamic.go

```diff
@@ -0,0 +1,11 @@
+//go:build dynamic
```

Comment:
Please add license headers for all files.

Use the following commands to format:

```shell
docker run -it --rm -v $(pwd):/github/workspace ghcr.io/korandoru/hawkeye-native format
```